mne/io/brainvision/brainvision.py

```diff
@@ -451,12 +463,13 @@ def _get_vhdr_info(vhdr_fname, eog, misc, scale):
         the header file. Example: {'FC1': 'nV'}
     """
     scale = float(scale)
-    ext = op.splitext(vhdr_fname)[-1]
-    if ext != '.vhdr':
+    ext = op.splitext(hdr_fname)[-1]
+    ahdr_format = True if ext == '.ahdr' else False
```

Comment:
An idea for a simplification. The parens are not strictly necessary but it makes it clearer that it's not a multiple assignment, but rather a single assignment involving a comparison / conversion to bool
```suggestion
    ahdr_format = (ext == '.ahdr')
```